classes/article/PublishedArticleDAO.inc.php

```diff
@@ -154,12 +232,14 @@ function getPublishedArticlesByJournalId($journalId = null, $rangeInfo = null, $
 				' . $this->getFetchColumns() . '
 			FROM	published_submissions ps
 				LEFT JOIN submissions s ON ps.submission_id = s.submission_id
+				LEFT JOIN submission_settings ss ON ss.submission_id = ps.submission_id
```

Comment:
shouldn't we then consider DISTINCT because we could get several submission_settings because of the revisions? Also elsewhere?